app/src/main/java/io/nekohasekai/sagernet/vpn/repositories/AppRepository.kt

```diff
@@ -384,7 +384,7 @@ object AppRepository {
         }
     }
 
-    public suspend fun silentUrlTestAsync() = withContext(Dispatchers.IO) {
+    suspend fun silentUrlTestAsync() = withContext(Dispatchers.IO) {
```

Comment:
It's still there! I said totally remove silentUrlTestAsync() from AppRepository!